openstack_sdk/common.py

```diff
@@ -92,6 +92,3 @@ def create(self):
 
     def delete(self):
         raise NotImplementedError()
-
-    def update(self, new_config=None):
```

Comment:
Do we not need update action?

Reply:
@0lvin There are a lot of resources that do not support update, so this force us to re-implement this in all resources that do not have update operation, so I just remove this method and keep it only on subclasses that has update method without having it on super class